wiki/Tournaments/osu!_Riverside/SWEAT/en.md

```diff
@@ -0,0 +1,122 @@
+---
+tags:
+- Fall
+- Highlander Con
+- LAN
+- osu! Riverside
+- Riverside
+- SWEAT
+- UCR
+---
+
+# Standard/Mania Weekend End of Autumn Tournament
+
+![SWEAT banner](img/banner.png)
```

Comment:
Is this image too big? 1.03 MB might be pushing it

Reply:
yes